### DISCUSSION TITLE
zoom feature in the IDE

### DISCUSSION THREAD

**Discussion by mustafa9514** [apache/netbeans#5803] [General]
View > View > Zoom

I want such a feature. It is very beneficial for the eyes. Can you add the ability to enlarge the entire screen at once?

**Comment by troizet** [comment]
You can try using `Alt + Shift + Mouse wheel`

https://user-images.githubusercontent.com/9607501/231073537-18637237-1768-4e3b-afe7-fc0db18494ab.mp4

**Reply by Chris2011** [reply]
We should also think about programmatically resizing the whole Window which is not the Editor. For presentations it is very helpful. Maybe we can do this with FlatLaf properties. For other Look and feel I dunno. Maybe update all fonts on scroll could be possible.
